.github/workflows/release-weekly.yml

```diff
@@ -23,4 +23,5 @@ jobs:
         ZAP_RELEASE: 1
         ZAP_JAVA_VERSION: 11
         ZAP_WEEKLY_ADDONS_NO_TEST: ${{ vars.ZAP_WEEKLY_ADDONS_NO_TEST }}
+        ZAP_WEEKLY_QUIET: ${{ vars.ZAP_WEEKLY_QUIET }}
       run: ./gradlew "-Dorg.gradle.jvmargs=-Xmx2g -XX:MaxMetaspaceSize=512m" :zap:createWeeklyRelease
```

Comment:
Have any of you checked this lately? To know if 2G is lots or if we're close to that?

Reply:
Should be enough for the add-ons included in the weekly (not building all of them like in zap-extensions).